### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by williamlifaith**
1.add a folder lang to contain all i18n files; and rename the file cn.js to i18n-cn.js
2.add a folder service and put i18n folder in it

### CODE REVIEW

File: src/components/first-page/lang/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './lang'
```
Comment (lanshunfang) [inline src/components/first-page/lang/index.js]: What is this? Where is lang file or folder? Require knowledge of relative path.

File: src/main.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import Vue from 'vue'
 import App from './App'
 import router from './router'
-import i18n from './i18n/i18n'
+import i18n from './service/i18n/i18n'
```
Comment (lanshunfang) [inline src/main.js]: no need for two i18n, just put i18n.js immediately under `./service` folder.

File: src/service/i18n/i18n.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-redeclare,indent,no-unused-vars */
+import Vue from 'vue'
+import VueI18n from 'vue-i18n'
+ import {LANG_EN} from '../../components/first-page/lang/i18n-en'
```
Comment (lanshunfang) [inline src/service/i18n/i18n.js]: Should put` index.js` inside `components/first-page/lang` and `components/first-page/` and `components`. Import files from it.

The line should be like:
```javascript
import {LANG_EN, LANG_CN, LANG_KO} from '../../components'
```

File: src/components/first-page/lang/i18n-en.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const LANG_EN = {
```
Comment (lanshunfang) [inline src/components/first-page/lang/i18n-en.js]: The file should not have LANG_EN. It should only have:

```javascript
export default {
  hello: 'hello world'
} 
```

Comment (lanshunfang) [reply src/components/first-page/lang/i18n-en.js]: Let's fix this this afternoon.

**Comment by williamlifaith** [comment]
Sorry I think this is not the right vision.

I will update it soon!

**Comment by williamlifaith** [comment]
Please check it again!

Thanks!

**Review by lanshunfang — APPROVED** [review]

**Pull request merged**